Gems/Multiplayer/Code/CMakeLists.txt

```diff
@@ -35,6 +34,89 @@ ly_add_target(
         *.AutoPackets.xml,AutoPackets_Source.jinja,$path/$fileprefix.AutoPackets.cpp
         *.AutoPackets.xml,AutoPacketDispatcher_Header.jinja,$path/$fileprefix.AutoPacketDispatcher.h
         *.AutoPackets.xml,AutoPacketDispatcher_Inline.jinja,$path/$fileprefix.AutoPacketDispatcher.inl
+)
+
+ly_add_target(
+    NAME Multiplayer.Client.Static STATIC
+    NAMESPACE Gem
+    FILES_CMAKE
+        multiplayer_files.cmake
+    INCLUDE_DIRECTORIES
+        PRIVATE
+            ${pal_source_dir}
+            Source
+            .
+        PUBLIC
+            Include
+    COMPILE_DEFINITIONS
+        PUBLIC
+            AZ_TRAIT_CLIENT_ENABLED=1
+            AZ_TRAIT_SERVER_ENABLED=0
+    BUILD_DEPENDENCIES
+        PUBLIC
+            Gem::Multiplayer.AutoGen.Static
+        PRIVATE
+            Gem::EMotionFXStaticLib
+            Gem::PhysX.Static
+    AUTOGEN_RULES
+        *.AutoComponent.xml,AutoComponent_Header.jinja,$path/$fileprefix.AutoComponent.h
+        *.AutoComponent.xml,AutoComponent_Source.jinja,$path/$fileprefix.AutoComponent.cpp
+        *.AutoComponent.xml,AutoComponentTypes_Header.jinja,$path/AutoComponentTypes.h
+        *.AutoComponent.xml,AutoComponentTypes_Source.jinja,$path/AutoComponentTypes.cpp
+)
+
+ly_add_target(
+    NAME Multiplayer.Server.Static STATIC
+    NAMESPACE Gem
+    FILES_CMAKE
+        multiplayer_files.cmake
+    INCLUDE_DIRECTORIES
+        PRIVATE
+            ${pal_source_dir}
+            Source
+            .
+        PUBLIC
+            Include
+    COMPILE_DEFINITIONS
+        PUBLIC
+            AZ_TRAIT_CLIENT_ENABLED=0
+            AZ_TRAIT_SERVER_ENABLED=1
+    BUILD_DEPENDENCIES
+        PUBLIC
+            Gem::Multiplayer.AutoGen.Static
+        PRIVATE
+            Gem::EMotionFXStaticLib
+            Gem::PhysX.Static
+    AUTOGEN_RULES
+        *.AutoComponent.xml,AutoComponent_Header.jinja,$path/$fileprefix.AutoComponent.h
+        *.AutoComponent.xml,AutoComponent_Source.jinja,$path/$fileprefix.AutoComponent.cpp
+        *.AutoComponent.xml,AutoComponentTypes_Header.jinja,$path/AutoComponentTypes.h
+        *.AutoComponent.xml,AutoComponentTypes_Source.jinja,$path/AutoComponentTypes.cpp
+)
+
+ly_add_target(
+    NAME Multiplayer.Unified.Static STATIC
+    NAMESPACE Gem
+    FILES_CMAKE
+        multiplayer_files.cmake
+    INCLUDE_DIRECTORIES
+        PRIVATE
+            ${pal_source_dir}
+            Source
+            .
+        PUBLIC
+            Include
+    COMPILE_DEFINITIONS
+        PUBLIC
+            AZ_TRAIT_CLIENT_ENABLED=1
+            AZ_TRAIT_SERVER_ENABLED=1
+    BUILD_DEPENDENCIES
+        PUBLIC
+            Gem::Multiplayer.AutoGen.Static
+        PRIVATE
+            Gem::EMotionFXStaticLib
+            Gem::PhysX.Static
+    AUTOGEN_RULES
```

Comment:
This is only added a dependency of the `Multiplayer` target and that target is not used by anything.

So both this target and `Multiplayer` targets should removed